### PR TITLE
Add a test for 400ZR transceiver low power mode setting.

### PR DESCRIPTION
**Pull request by ahsaanyousaf**
Add a test for 400ZR transceiver low power mode setting.

### CODE REVIEW

File: feature/platform/transceiver/zr_low_power_mode_test/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  * In Low Power Mode (characteristic of all MSM steady states except
+    ModuleReady) the maximum module power consumption is defined in the form
+    factor-specific hardware specification.
+  * In High Power Mode (characteristic of the MSM state ModuleReady) the
+    implementation dependent maximum module power consumption is advertised in
+    the MaxPower Byte 00h:201. More details in the CMIS link below.
+
+Link to CMIS:
+https://www.oiforum.com/wp-content/uploads/CMIS5p0_Third_Party_Spec.pdf
+
```
Comment (dplore) [inline feature/platform/transceiver/zr_low_power_mode_test/README.md:21]: Add a test topology section, for example: https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md?plain=1#L25-L27

File: feature/platform/transceiver/zr_low_power_mode_test/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    
+    *   /interfaces/interface/config/enabled
+
+*   In low power mode the module's Management interface should be available,
+    entire paged management memory should be accessible, and host may configure
+    the module.
+*   Low power mode represents the situation where the management interface is
+    fully initialized and operational while the device is still in Low Power
+    Mode. During this state, the host may configure the module using the
+    management interface to read from and write to the management Memory Map.
```
Comment (dplore) [inline feature/platform/transceiver/zr_low_power_mode_test/README.md:44]: This looks somewhat duplicated?

File: feature/platform/transceiver/zr_low_power_mode_test/README.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    *   /platform/components/component/state/hardware-version
+    *   /platform/components/component/state/firmware-version
+
+*  With module in low power mode verify that the module laser is squelched
+   and it is no longer able to report output-power under the following OC
+   paths.
+    *   /components/component/optical-channel/state/output-power/instant
+    *   /components/component/optical-channel/state/output-power/avg
+    *   /components/component/optical-channel/state/output-power/min
+    *   /components/component/optical-channel/state/output-power/max
```
Comment (dplore) [inline feature/platform/transceiver/zr_low_power_mode_test/README.md:67]: Is there a specific value or range you except to see here?  Should the device return nothing, as in the paths do not exist?  Or should they return with -99?

**Review by dplore — CHANGES_REQUESTED** [review]

File: feature/platform/transceiver/zr_low_power_mode_test/README.md (hunk 2 above)
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_low_power_mode_test/README.md:44]: Deleted the duplicate info

File: feature/platform/transceiver/zr_low_power_mode_test/README.md (hunk 1 above)
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_low_power_mode_test/README.md:21]: Done

File: feature/platform/transceiver/zr_low_power_mode_test/README.md (hunk 3 above)
Comment (ahsaanyousaf) [reply feature/platform/transceiver/zr_low_power_mode_test/README.md:67]: Updated, thanks

**Review by dplore — APPROVED** [review]

File: feature/platform/transceiver/zr_low_power_mode_test/README.md [hunk 4]
```diff
@@ -29,19 +29,23 @@ https://www.oiforum.com/wp-content/uploads/CMIS5p0_Third_Party_Spec.pdf
...
       * Both transceivers are tuned to a valid centre frequency
         example 193.1 THz.
-*   With the ZR link established as explained above, verify that the
-    following ZR transceiver OC path when set to False is able to move to the
-    low power mode as defined in the CMIS.
+## Testbed Type
+*   Typical test setup for this test is a DUT1 with 2 ports to 2 ATE ports or 2
+    ports to a second DUT2. For most tests this setup should be sufficient.
+    Ref: [Typical ATE<>DUT Test bed](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+*   A and Z ends of  the link should have same 400ZR PMD. For this test a
```
Comment (ejbrever) [inline feature/platform/transceiver/zr_low_power_mode_test/README.md:36]: nit, extra space after "of"

File: feature/platform/transceiver/zr_low_power_mode_test/README.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    single DUT ZR port connected to a single ZR ATE port is also sufficient. 
+
+Once the ZR link is estabished proceed with the following:
+*   Verify that the following ZR transceiver OC path when set to False is able
+    to move to the low power mode as defined in the CMIS.
+    
+    *   /interfaces/interface/config/enabled
+
+*   In low power mode the module's Management interface should be available,
```
Comment (ejbrever) [inline feature/platform/transceiver/zr_low_power_mode_test/README.md:45]: optional, but are we going too deep into CMIS jargon here when this is an OpenConfig test? Could we essentially delete this bullet and the following one without losing any intent of this test?

**Pull request merged**